fastcov.py

```diff
@@ -955,6 +983,9 @@ def parseArgs():
     parser.add_argument('-p', '--dump-statistic', dest="dump_statistic", action="store_true", help="Dump total statistic at the end")
     parser.add_argument('-v', '--version', action="version", version='%(prog)s {version}'.format(version=__version__), help="Show program's version number and exit")
 
+    parser.add_argument('-gps', '--gcov_prefix_strip', dest="gcov_prefix_strip", action="store", default=0, type=int, help="The number of initial directory names to strip off the absolute paths in the object file.")
```

Comment:
Since `GCOV_PREFIX_STRIP` is only meaningful with `0` or `1`, I recommend using using bool:
```python
    parser.add_argument('-gps', '--gcov_prefix_strip', dest="gcov_prefix_strip", action="store_true", help="The number of initial directory names to strip off the absolute paths in the object file.")
```

Reply:
GCOV_PREFIX_STRIP is the number of folder levels to strip, so it's not 0 or 1.
With GCOV_PREFIX_STRIP set to 2, /home/lapointejp/git/project would be come ./git/project and if you set it to 1 it would become ./lapointejp/git/project.